dvc/commands/ls/__init__.py

```diff
@@ -41,8 +46,8 @@ def run(self):
             if self.args.json:
                 ui.write_json(entries)
             elif entries:
-                entries = _prettify(entries, with_color=True)
-                ui.write("\n".join(entries))
+                entries = _prettify(entries, with_color=True, with_size=self.args.size)
+                ui.table(entries)
```

Comment:
Kudos to @skshetry for `ui.table`, feels really nice to use.

Reply:
Note that `ui.table()` can be slow when we are printing a million lines. 😄 
So we were using raw prints and `"\n".join()` to print everything at once.

Reply:
@skshetry Good point. I guess we can keep that optimization in the simplest case. I'll take a look.